### PR TITLE
Blinky LED + Baked in Assets

### PR DESCRIPTION
**Pull request by FryDay**
If we go with this method, before merging we should either move the kongtext license to it's own folder or figure out a clean way to exclude it so we don't bake it into the binary unnecessarily.

### CODE REVIEW

**Comment by kbembedded** [comment]
Probably safe to move the license text to root dir or src/ or even just add it to readme. According to the actual font license it says "...give credit somewhere in the font files to ..." and the .ttf already credits codeman38. I would say the license file could just be removed even. Thoughts?

**Comment by FryDay** [comment]
I'd say we are safe to remove it and maybe put something like this in the main readme:
`Kongtext Font by [ codeman38 | cody@zone38.net | http://www.zone38.net/ ]`

**Comment by kbembedded** [comment]
Make it so.

File: src/ui.c [hunk 1]
```diff
@@ -27,15 +33,29 @@ SDL_Texture* lcd_tex = NULL;
 SDL_Rect lcd_srcRect = { 0, 0, 320, 240 };
 SDL_Rect lcd_dstRect = { 0, 0, LOGICAL_WIDTH, LOGICAL_HEIGHT };
 
+// LED
+#define UI_LED_IMAGE_SIZE 32
+extern const uint8_t led_png[];
+extern const uint16_t led_png_size;
```
Comment (CaptainNic) [inline src/ui.c:39]: Why are these extern if they're defined in the header file?

Comment (FryDay) [reply src/ui.c:39]: Good point, I had this 2 different ways and then apparently went half-way with both. Fixing.

File: src/CMakeLists.txt [hunk 2]
```diff
@@ -3,15 +3,39 @@ add_executable(msemu
 	flashops.c
 	main.c
 	msemu.c
-	ui.c
+    ui.c
+    ${CMAKE_BINARY_DIR}/fonts.c
+    ${CMAKE_BINARY_DIR}/images.c
```
Comment (CaptainNic) [inline src/CMakeLists.txt:8]: Whitespace

**Review by CaptainNic — CHANGES_REQUESTED** [review]

**Comment by kbembedded** [comment]
We doin tabs in CMakeLists.txt now? toplevel uses spaces lol

**Review by CaptainNic — APPROVED** [review]
LGTM

**Comment by kbembedded** [comment]
Tests fine when binary is moved to its own dir.

**Pull request merged**